projects/graph/src/graph.py

```diff
@@ -20,3 +22,10 @@ def add_edge(self, vertex1, vertex2):
             self.vertices[vertex2].add(vertex1)
         else:
             raise Exception("Edges to nonexistent vetices")
+
+    def bredth_first_traversal(self, start_node):
+        queue = Queue()
```

Comment:
If you're going to use visited to just print out the visited nodes in order i suggest using a list since they have an index.

Reply:
Based off the code you commented on, do you want me to not use the queue class, instead just use a list? Visited is already a list on line 28.

Reply:
You can use the queue to help guide the bft, I was referring to your visited set.